SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -33,8 +35,13 @@ void GatherClientInterestDistances()
 	TMap<UClass*, float> DiscoveredInterestDistancesSquared;
 	for (TObjectIterator<UClass> It; It; ++It)
 	{
-		if (It->HasAnySpatialClassFlags(SPATIALCLASS_ServerOnly))
+		if (It->HasAnySpatialClassFlags(SPATIALCLASS_ServerOnly | SPATIALCLASS_NotSpatialType))
+		{
+			continue;
+		}
+		if (It->HasAnyClassFlags(CLASS_NewerVersionExists))
```

Comment:
Worth putting this in an #if WITH_EDITOR block?

Reply:
will do as part of https://improbableio.atlassian.net/browse/UNR-1649

Reply:
more detail. discussed caching with @Vatyx, planning to not add caching for the class hierarchy stuff in this change when not running PIE, and to remove caching for the net cull distance stuff in the same circumstance. Should clean up the caching code and remove a class of edge case.